mg-criticality/make_model.py

```diff
@@ -303,7 +303,7 @@ def make_model(self):
     def execute(self, quiet=True):
         returncode = openmc.run(output=(not quiet))
         spfile = 'statepoint.' + str(self.batches) + '.h5'
-        sp = openmc.StatePoint(spfile)
-        self.keff = sp.k_combined[:]
+        sp = openmc.StatePoint(spfile, autolink=False)
+        self.keff = sp.k_combined
 
         return returncode
```

Comment:
`openmc.run` doesn't actually return a returncode anymore but will raise `subprocess.CalledProcessError`, so this should be changed to catch the exception and return a non-zero value (or let the exception propagate and catch it elsewhere).